tree/dataframe/inc/ROOT/RDF/ColumnReaders.hxx

```diff
@@ -41,24 +41,31 @@ void CheckDefine(RDFDetail::RDefineBase &define, const std::type_info &tid);
 \class ROOT::Internal::RDF::RColumnReaderBase
 \ingroup dataframe
 \brief Pure virtual base class for all column reader types
-\tparam T The type of the column
 
 This pure virtual class provides a common base class for the different column reader types, e.g. RTreeColumnReader and
 RDSColumnReader.
 **/
-template <typename T>
 class RColumnReaderBase {
 public:
    virtual ~RColumnReaderBase() = default;
+
    /// Return the column value for the given entry. Called at most once per entry.
```

Comment:
Maybe add `\tparam` and `\param` docstrings, or is it only internal? I guess it's nice to have at least the base fully documented.